static/sass/components/_post-header.scss

```diff
@@ -2,22 +2,58 @@
 // ===========
 
 .title {
-  @include title;
   margin: size('gutter') auto;
   max-width: size('full-content');
   padding: 0 size('gutter');
 
-  &:first-child {
+  .no-image & {
     @include text-shadow;
+    @include title;
   }
+}
+
+// Titles on pages that live inside title wrapper
+.title-main {
+  @include text-shadow;
+  @include title;
+}
+
+// Titles on pages that don't have title wrapper
+.page-title {
```

Comment:
oops, I was trying to add a new class in order to not have the title-main styles applying to the .title since one is a wrapper (that should inherit position only) and the other is the element itself so it would need the positioning AND the font-styles. I have to look into how to handle both with the revisions.